PlayerInfo/PlayerInfo.cpp

```diff
@@ -130,5 +174,217 @@ namespace Plugin {
         }
     }
 
+    uint32_t PlayerInfo::getSinkAtmosCapability (const JsonObject& parameters, JsonObject& response)
+    {   //sample servicemanager response:
+        LOGINFOMETHOD();
+        bool success = true;
+        dsATMOSCapability_t atmosCapability;
+        try
+        {
+            device::AudioOutputPort aPort = device::Host::getInstance().getAudioOutputPort("HDMI0");
+            if (aPort.isConnected()) {
+                aPort.getSinkDeviceAtmosCapability (atmosCapability);
+                response["atmos_capability"] = (int)atmosCapability;
```

Comment:
For Metrological the Coding rule is applicable: in C++ code we use C++ casting. C casting does not explicitly state the type of casting and the severity of the cast can not be determined.